third_party/terraform/utils/bootstrap_utils_test.go

```diff
@@ -112,6 +131,10 @@ func BootstrapKMSKeyInLocation(t *testing.T, locationID string) bootstrappedKMS
 	}
 }
 
+func BootstrapKMSKeyInLocation(t *testing.T, locationID string) bootstrappedKMS {
```

Comment:
nit: do you mind moving this above `BootstrapKMSKeyWithPurposeInLocation`? I'd like to group the overloads together so it's immediately clear how the main fn could have been called.

Reply:
Pushed changes.